tests/download.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
-if [ $# -ne 1 ]; then
+if [ $# -le 1 ]; then
```

Comment:
In the old code, 1 argument was ok (while more than 1 should have been but wasn't). With this test 1 argument would fail. This should be `-lt` not `-le`.